backends/occa/ceed-occa-operator.c

```diff
@@ -503,32 +495,21 @@ static int CeedOperatorApply_Occa(CeedOperator op,
 
   // Output restriction
   for (CeedInt i=0; i<qf->numoutputfields; i++) {
+    // Restore evec
+    ierr = CeedVectorRestoreArray(data->Evecs[i+data->numein],
+                                    &data->Edata[i + qf->numinputfields]); CeedChk(ierr);
     // Active
     if (op->outputfields[i].vec == CEED_VECTOR_ACTIVE) {
-      // Restore evec
-      ierr = CeedVectorRestoreArray(data->Evecs[i+data->numein],
-                                    &data->Edata[i + qf->numinputfields]); CeedChk(ierr);
       // Zero lvec
-      ierr = CeedVectorGetArray(outvec, CEED_MEM_HOST, &vec_temp); CeedChk(ierr);
-      for (CeedInt j=0; j<outvec->length; j++)
-        vec_temp[j] = 0.;
-      ierr = CeedVectorRestoreArray(outvec, &vec_temp); CeedChk(ierr);
+      ierr = CeedVectorSetValue(outvec, 0.0); CeedChk(ierr);
```

Comment:
When we enable SetValue/SetArray using GPU memory (Issue #119), this should speed up nicely.

Reply:
Should we take this statement to mean you are not currently working on the backend, therefore you would appreciate any maintenance to update backends/occa as well?

As a broader communication matter, should we generally interpret lack of a WIP branch or PR to mean that there is no relevant work in progress (so we don't need to worry about creating merge conflicts)? Maybe we should chat about this on the telecon.